docs/install-csi-driver-master.md

```diff
@@ -1,35 +1,33 @@
-## Install azuredisk CSI driver development version on a Kubernetes cluster
-
-### Installation with kubectl
+## Install CSI driver development version on a Kubernetes cluster
 
+### Install by kubectl
 #### Linux
+ - option#1
+```console
+curl -skSL https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy/install-driver-standalone.sh | bash -s --
+```
 
+ - option#2
 ```console
 kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy/csi-azuredisk-driver.yaml
 kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy/crd-csi-node-info.yaml
 kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy/rbac-csi-azuredisk-controller.yaml
 kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy/csi-azuredisk-controller.yaml
 kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy/csi-azuredisk-node.yaml
+```
 
-# skip below yaml configurations if snapshot feature(only available from v1.17.0) will not be used
+ - install snapshot components
+> skip below yaml configurations if snapshot feature(only available from v1.17.0) will not be used
+```
 kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy/crd-csi-snapshot.yaml
 kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy/rbac-csi-snapshot-controller.yaml
 kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy/csi-snapshot-controller.yaml
 ```
 
 #### Windows
-
-Windows CSI driver requires [CSI Proxy](https://github.com/kubernetes-csi/csi-proxy) to be installed on the system. Please use the 
```

Comment:
@andyzhangx - without these instructions the overall flow might not be clear to user. And could result in many users deploying csi drivers on windows without having csi proxy installed. What do you say ?

Reply:
good point, I removed this windows section in this page, and move to https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/deploy/example/windows, that would be more clear.